cql/src/main/java/io/stargate/cql/CqlActivator.java

```diff
@@ -95,13 +98,20 @@ private static Config makeConfig() {
 
       Integer cqlPort = Integer.getInteger("stargate.cql_port", 9042);
 
-      Config c = new Config();
+      Config c;
+      String cassandraConfigPath = System.getProperty("stargate.unsafe.cassandra_config_path", "");
```

Comment:
I don't see how this change relates to `Fix client encryption`. Can you elaborate more on it?

Reply:
It allows configuring the client TLS using a `cassandra.yaml` file. But this property name is already taken: https://github.com/stargate/stargate/pull/602